client/src/routes/AppRouts.tsx

```diff
@@ -0,0 +1,29 @@
+import React ,{ Suspense , lazy } from "react"
+import { BrowserRouter , Route, Routes } from "react-router-dom"
+
+
+const Login = lazy(() => import('../views/login/Login'))
+const Dashboard = lazy(() => import("../views/dashboard/Dashboard"))
+const RoomCreate = lazy(() => import("../views/roomCreat/RoomCreate"))
+const RoomEdit = lazy(() => import("../views/roomEdit/RoomEdit"))
```

Comment:
better approach to add index.ts 
that export the file then the import will be shorter 
```suggestion
const Login = lazy(() => import('../views/login'))
const Dashboard = lazy(() => import("../views/dashboard"))
const RoomCreate = lazy(() => import("../views/roomCreat"))
const RoomEdit = lazy(() => import("../views/roomEdit"))
```

you need to add index.ts in each folder -
`export { default } from './login'` - for example